style.css

```diff
@@ -1,8 +1,57 @@
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+@import url('https://fonts.googleapis.com/css2?family=Inconsolata&family=Ubuntu&display=swap');
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+:root {
+  /* COLORS */
+  --color-primary: #ff9800;
+  --color-header-text: #283557;
+  --color-main-text: #d2d4dc;
+  --color-background-header: rgb(180, 186, 180);
+  --color-background-main: #2d2f32;
+  --color-background-card: #3c3e44;
+
+  /* FONT */
+  --font-family-headline: 'Inconsolata', monospace;
+  --font-family-text: 'Ubuntu', sans-serif;
+
+  /* BORDERS */
+  --border-radius: 15px;
+}
+
+body {
+  font-family: var(--font-family-text);
+  margin: 0;
+  background-color: var(--color-background-main);
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: var(--font-family-headline);
+}
+
+header {
```

Comment:
Be careful using element selectors here. You might want to use `header` in other components too.